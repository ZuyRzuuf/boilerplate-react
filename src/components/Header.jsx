import React from 'react';

const Header = () => {
    return (
        <div>Backend/Frontend boilerplate without PHP</div>
    );
};

export default Header;
