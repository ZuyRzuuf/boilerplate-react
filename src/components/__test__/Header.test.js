import React from 'react';

import Header from '../Header';

describe('A suite', function() {
    it('renders correctly', () => {
        const wrapper = shallow(
            <Header />
        );
        expect(wrapper).toMatchSnapshot();
    });
});
