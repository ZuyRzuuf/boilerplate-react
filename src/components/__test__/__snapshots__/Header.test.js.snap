// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A suite renders correctly 1`] = `
<div>
  Backend/Frontend boilerplate without PHP
</div>
`;
